build.js

```diff
@@ -147,5 +271,7 @@ async function buildMod(args, info) {
 }
 
 if (module === require.main) {
-	main().catch(err => { console.log(err) });
+	(async () => {
+		await main().catch(err => { console.log(err) });
+	})();
```

Comment:
What does this async warp achieve?

Reply:
Remnant from extra tooling I run in my local environment, i could revert it but the diff is cleaner like this